site/content/docs/0.0/utilities/display.md

```diff
@@ -1,11 +1,116 @@
 ---
 layout: docs
 title: Display property
-description: Quickly and responsively toggle the display value of components and more with our display utilities. Includes support for some of the more common values, as well as some extras for controlling display when printing.
+description: Quickly <!--and responsively -->toggle the display value of components and more with our display utilities. Includes support for some of the more common values, as well as some extras for controlling display when printing.
 group: utilities
 aliases:
   - "/docs/utilities/display/"
 toc: true
 ---
 
-{{< callout-soon "page" >}}
+## How it works
+
+Change the value of the [`display` property](https://developer.mozilla.org/en-US/docs/Web/CSS/display) with our <!--responsive -->display utility classes. We purposely support only a subset of all possible values for `display`. Classes can be combined for various effects as you need.
+
+## Notation
+
+Display utility classes that apply to all breakpoints<!-- [breakpoints]({{< docsref "/layout/breakpoints" >}}), from `xs` to `xxl`-->, have no breakpoint abbreviation in them. This is because those classes are applied from `min-width: 0;` and up, and thus are not bound by a media query. The remaining breakpoints, however, do include a breakpoint abbreviation.
```

Comment:
```suggestion
<!--Display utility classes that apply to all [breakpoints]({{< docsref "/layout/breakpoints" >}}), from `xs` to `xxl`, have no breakpoint abbreviation in them. This is because those classes are applied from `min-width: 0;` and up, and thus are not bound by a media query. The remaining breakpoints, however, do include a breakpoint abbreviation.-->
```

Reply:
I felt like we could still have this paragraph since we will have breakpoints, we just "don't know" yet what will be those breakpoints.

Reply:
I was suggesting this because this paragraph talks about "breakpoint abbreviation"s, that are basically xs, md, lg, etc.